src/controllers/EscapePodController.test.ts

```diff
@@ -23,6 +23,29 @@ describe(
 
     afterEach(() => sandbox.restore());
 
+    it(
+      "getAll should call EscapePodsService and return 200", async () => {
+
+        // Given
+        const escapePodsMock = [ getEscapePodMock(), getEscapePodMock(), getEscapePodMock() ];
```

Comment:
You'd want to test that `fakeRes.json` has escape pod data. You can either do that in this test or create a separate test for it.